pkg/backend/aws/backend.go

```diff
@@ -189,6 +188,23 @@ func NewAWSBackend(
 	return b, nil
 }
 
+func (b *Backend) getComputeBackend(ctx context.Context, happyConfig *config.HappyConfig) (interfaces.ComputeBackend, error) {
+	var computeBackend interfaces.ComputeBackend
+	var err error
+	if happyConfig.TaskLaunchType() == config.LaunchTypeK8S {
+		computeBackend, err = NewK8SComputeBackend(ctx, happyConfig, b)
+		if err != nil {
+			return nil, errors.Wrapf(err, "unable to connect to k8s backend")
+		}
+	} else {
+		computeBackend, err = NewECSComputeBackend(ctx, happyConfig, b)
+		if err != nil {
+			return nil, errors.Wrapf(err, "unable to connect to ecs backend")
+		}
+	}
+	return computeBackend, nil
```

Comment:
```suggestion
if happyConfig.TaskLaunchType() == config.LaunchTypeK8S {
	return NewK8SComputeBackend(ctx, happyConfig, b)
}

return NewECSComputeBackend(ctx, happyConfig, b)
```

simplify your nested blocks and if/elses

https://github.com/uber-go/guide/blob/master/style.md#unnecessary-else
https://github.com/uber-go/guide/blob/master/style.md#reduce-nesting